pkg/operator/observe_config.go

```diff
@@ -99,6 +103,25 @@ func (c ConfigObserver) sync() error {
 		unstructured.SetNestedStringSlice(observedConfig, etcdURLs, "storageConfig", "urls")
 	}
 
+	internalRegistryHostName := ""
+	imageConfigClient, err := configv1.NewForConfig(c.clientConfig)
+	if err != nil {
+		return err
+	}
+	configImageList, err := imageConfigClient.Images().List(metav1.ListOptions{})
```

Comment:
add a TODO to look for a particular name

Reply:
So the intent here is to fetch the ImageConfig on every sync, rather than consume it from the event that triggered the sync?

Reply:
(and i assume there's still work coming to setup an informer on the ImageConfig so the sync gets triggered when ImageConfig changes?)